src/hooks/Bonds.ts

```diff
@@ -1,13 +1,13 @@
-import { useSelector } from "react-redux";
 import { useEffect, useState } from "react";
 import orderBy from "lodash/orderBy";
+import { useAppSelector } from ".";
 
 export const makeBondsArray = (
-  ohmDaiBondDiscount,
-  ohmFraxLpBondDiscount,
-  daiBondDiscount,
-  fraxBondDiscount,
-  ethBondDiscount,
+  ohmDaiBondDiscount?: number | undefined,
```

Comment:
shouldn't the default be 0, not undefined?

Reply:
yeah, not sure why I put undefined there, will remove that, the params are optional because of how the function is being used.